### ISSUE TITLE
Questions regarding i18n rule

### ISSUE THREAD

**Issue by andrefilimono** [angular-eslint/angular-eslint#298]
1) It would be really useful to be able to skip some simple HTML items that in most cases shouldn't be translated:

```html
<span>&nbsp;</span>
<span>$</span>
```

I tried to filter out those texts using `boundTextAllowedPattern` but without success and end up rewriting such items as follows:

```html
<ng-container>&nbsp;</ng-container>
<ng-container>$</ng-container>
```

and setting `ignoreTags: ['ng-container']`. It's not a good solution, though.

I wonder whether it's possible to achieve that with `boundTextAllowedPattern` or there should be another new option like `ignoreTexts`?

2) By setting `ignoreTags` I expect that those tag would be skipped from all checks, but it's not so.

Say I have the following code:

```html
<my-component size="s"></my-component>
```

and this config:

```js
ignoreTags: ['my-component']
```

ESLint will report a problem saying `Attribute size has no corresponding i18n attribute.` 

Is it an expected behavior or a bug?

**Comment by DerZyklop** [comment]
@andrefilimono there is a property called boundTextAllowedPattern. I think it expects a RegEx. I think that could be a possible solution.

**Comment by JamesHenry** [comment]
@rafaelss95 Are you able to help out here?

**Comment by rafaelss95** [comment]
From @janousek on #329:

> Currently there is a pattern ("boundTextAllowedPattern" option) to filter out non-textual content. But it works only for bound text (for example: `<span>+{{data_from_backend}}</span>`). 
https://github.com/angular-eslint/angular-eslint/blob/eeb49d491bb92e470f505a699ef3e12963ff351b/packages/eslint-plugin-template/src/rules/i18n.ts#L252

> But it is not applied to text nodes (for example: `<span>+</span>`).
https://github.com/angular-eslint/angular-eslint/blob/eeb49d491bb92e470f505a699ef3e12963ff351b/packages/eslint-plugin-template/src/rules/i18n.ts#L249

> So there is currently no way to filter out this content. There are currently two options:
> 1. Apply "checkBoundTextAllowedPattern" to the text nodes.
> 2. Add new option ("checkTextAllowedPattern") and apply it to the text nodes.

**Comment by rafaelss95** [comment]
Hello @JamesHenry, sorry for the delay. I must confess that I'm not familiar with Angular i18n and also with what all these options represent (unfortunately the code isn't that expressive and we don't have documentation for this rule yet 😞 ).

That said, for the first point I guess @janousek said it all and now I think we should decide which approach we should take: either create a new option called `textAllowedPattern` to check only non-bounded texts or create this option to check all types of texts and thus deprecating the `boundTextAllowedPattern`.

For the second, I have created the PR #387.

**Comment by JamesHenry** [comment]
Ok so I'll class this one as a bug fix which corresponds to the PR, if there is a new feature to be considered please could we spin it out into its own issue @andrefilimono?
